src/Saas.Admin/Saas.Admin.Service/Program.cs

```diff
@@ -35,6 +68,9 @@
 {
     app.UseSwagger();
     app.UseSwaggerUI();
+
+    // Show identity debug information during development. This should be kept off in production to comply with GDPR standards
+    IdentityModelEventSource.ShowPII = true;
```

Comment:
we need to be careful with this setting

Reply:
Thanks for the catch. I have only turned on in development on the permissions API. I'll add a check here too.